content/collections/destination-catalog/en/hubspot-event-streaming-v2.md

```diff
@@ -52,6 +53,19 @@ Keep these things in mind when sending events to HubSpot:
 7. Under *Map properties to destinations*, select the Event Properties you would like to send. The HubSpot identifier for this object must be at least one of the following: `User Token`, `Email` or `Object ID`.
 8. Under *Select additional properties*, select any extra event and user properties you want to send to HubSpot. If you don't select any properties here, Amplitude doesn't send any.
 9. When finished, enable the destination and click *Save*.
+10. Click *Create in HubSpot* if there are events and/or properties that are missing.
+11. Click *Continue*, verify your connection, and click *Finish*.
+
+### Troubleshooting
+If you manually created and then deleted a custom event in HubSpot, the internal name is locked and can't be reused. Here's a workaround:
```

Comment:
Might help to summarize what they will be doing, especially for users familiar with Amplitude. "As a workaround, you can create a custom event that references the original event, and use that instead."